esp/src/eclwatch/Utility.js

```diff
@@ -19,6 +20,13 @@
         ;
     }
 
+    function csvEncode(str) {
+        str = "" + str;
+        return str.replace(/"/g, '\"')
+                  .replace(/','/g, '\'')
+        ;
+    }
```

Comment:
Agree with your numeric point but, I dont think if Jobname is: `M'sJobName` we should be returning `"M'sJobName"`